contracts/supplementary/rebalancing-manager/BTCDaiRebalancingManager.sol

```diff
@@ -39,16 +39,17 @@ contract BTCDaiRebalancingManager {
 
     /* ============ Constants ============ */
 
-    uint256 constant PRICE_PRECISION = 100;
+    uint256 constant PRICE_PRECISION = 1;
     uint256 constant AUCTION_LIB_PRICE_DIVISOR = 1000;
+    
     // Equal to $1 
-    uint256 constant DAI_PRICE = 10**18;
+    uint256 constant DAI_PRICE = 10 ** 18;
     uint256 constant DAI_DECIMALS = 18;
     uint256 constant BTC_DECIMALS = 8;
-    uint256 constant DECIMAL_DIFF_MULTIPLIER = 10**10;
+    uint256 constant DECIMAL_DIFF_MULTIPLIER = 10 ** 10;
     uint256 constant SET_TOKEN_DECIMALS = 18;
     uint256 constant THIRTY_MINUTES_IN_SECONDS = 1800;
-    uint256 constant VALUE_TO_CENTS_CONVERSION = 10**16;
+    uint256 constant VALUE_TO_CENTS_CONVERSION = 10 ** 16;
```

Comment:
We have a new function in `CommonMath` called `safePower`. Seems fine in this file since everything is exponentiated by a constant... but would consider using it to ensure no overflows